libraries/ui/src/CourseCard.stories.tsx

```diff
@@ -1,39 +0,0 @@
-import type { Meta, StoryObj } from '@storybook/react';
-
-import { CourseCard } from './CourseCard';
-
-const meta = {
-  title: 'ui/CourseCard',
-  component: CourseCard,
-  tags: ['autodocs'],
-  parameters: {
-    // More on how to position stories at: https://storybook.js.org/docs/configure/story-layout
-    layout: 'centered',
-  },
-} satisfies Meta<typeof CourseCard>;
-
-export default meta;
-type Story = StoryObj<typeof meta>;
-
-export const Default: Story = {
-  args: {
-    title: 'Alignment',
-    description: 'AI systems are rapidly becoming more capable and more general. Despite AI’s potential to radically improve human society, there are still open questions about how we build AI systems that are controllable, aligned with our intentions and interpretable.',
-    courseType: 'In-depth course',
-    imageSrc: '/images/intro-course.png',
```

Comment:
This image was only used in this story, but the component itself is no longer used, I deleted it along with the image